spec/runners/unified.rb

```diff
@@ -69,11 +69,11 @@ def define_unified_spec_tests(base_path, paths, expect_failure: false)
                   test.assert_events
                 # HACK: other errors are possible and likely will need to
                 # be added here later as the tests evolve.
-                rescue Mongo::Error::OperationFailure, Unified::Error::UnsupportedOperation
+                rescue ArgumentError, Mongo::Error::OperationFailure, Unified::Error::UnsupportedOperation, Unified::Error::EntityMissing
```

Comment:
I don't love checking against `EntityMissing` but for the tests that give a change stream that doesn't exist and expect failure, this is what we raise. I'm not sure how else to test that.

Reply:
EntityMissing is probably OK but why is ArgumentError here?

Reply:
see my comment: https://github.com/mongodb/mongo-ruby-driver/pull/2577#discussion_r937035251